acquia_lift.libraries.yml

```diff
@@ -2,3 +2,7 @@ acquia-lift:
   css:
     state:
       assets/acquia_lift.css: {}
+  js:
+    assets/acquia_lift.js: {}
+  dependencies:
+    - core/jquery
```

Comment:
Oops, this still needs to become `core/jquery.once`.

It's only working by accident now, because something else is depending on that asset library.